db/corruption_test.cc

```diff
@@ -173,10 +188,48 @@ class CorruptionTest : public testing::Test {
     ASSERT_GE(max_expected, correct);
   }
 
+  void CorruptFile(const std::string& fname, int offset, int bytes_to_corrupt) {
```

Comment:
This is copied from testutil.cc?

Reply:
Where does this used?

Reply:
I guess this function was not there when I created this PR on my local repo last year...Let me remove it if possible.

Reply:
Weird, it is not used. Will remove it.